modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml

```diff
@@ -92,6 +92,45 @@
 		<fail if="gogoshell.timeout" message="The Gogo shell connection did not connect as expected." />
 	</target>
 
+	<target name="check-gogoshell-connection">
+		<get-database-upgrade-client-paths />
+
+		<echo file="${db.upgrade.client.home}/telnet-connection.sh">
+			#!/bin/sh
+
+			telnet localhost 11312
```

Comment:
Used port 11312 instead of 11311 because CI adds `module.framework.properties.osgi.console=11312` to portal-ext